packages/api/src/utils/billing.js

```diff
@@ -70,7 +86,7 @@ export async function getPaymentSettings (ctx) {
 }
 
 /**
- * @returns {import('./stripe').UserCustomerService & { userIdToCustomerId: Map<string,string> }}
+ * @returns {import('./stripe').UserCustomerService & import('./stripe').TermsOfServiceService & { userIdToCustomerId: Map<string,string> }}
```

Comment:
ooh can you instead just make a separate `createMockTermsOfServiceService` fn?
The only reason I added the extra `&` here before was to make the mock a little introspectable e.g. for some unit testing. I think it'd be nice to keep the two mock service types separate if possible (e.g. so consumers of these functions can do [ISP](https://en.wikipedia.org/wiki/Interface_segregation_principle))

Reply:
I did this. ✅